### DISCUSSION TITLE
[A-L] (14 marzo 2024) Esercizio 4

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#20] [Esercitazioni A-L]
Sia $G$ il grafo raffigurato in figura. Determinare il minimo numero di archi che devono essere eliminati da $G$ affinché $G$ ammetta
ordinamenti topologici. Una volta rimosso questo insime minimo di archi, determinare tutti gli ordinamenti topologici di $G$.

![image](https://github.com/sapienzastudentsnetwork/progalg2324/assets/40034429/d51df7da-088b-4533-bd24-8315f678a4c2)

**Comment by daromacarlo** [comment]
![Immagine 2024-03-17 183843](https://github.com/sapienzastudentsnetwork/progalg2324/assets/135441131/57590327-53d1-4b9f-9568-a23e9671d428)
![Immagine 2024-03-17 183907](https://github.com/sapienzastudentsnetwork/progalg2324/assets/135441131/75f4da0f-7f29-427f-9d74-1c7d8c317e47)

**Reply by IonutCicio** [reply]
Basta levare l'arco da $c$ a $d$ e non ci sono più cicli... e gli ordini topologici sono effettivamente 2

**Reply by daromacarlo** [reply]
@CuriousCI vero👍🏻

**Comment by niccolo-fato** [comment]
![Esercizio 1_240716_152932](https://github.com/user-attachments/assets/58991253-50d5-4dfe-9ab8-3b4083b8ef56)
